<launch>
 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="teleop" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.3" />
  </node>
<!-- rosserial node -->
<!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="3" type="int"/>
  <param name="axis_strafe" value="0" type="int"/>
  <param name="axis_save" value="2" type="int"/>
  <param name="scale_linear" value="1" type="double"/>
  <param name="scale_angular" value="1" type="double"/>
  <param name="scale_strafe" value="1" type="double"/>
  <param name="scale_save" value="1" type="double"/>
  <node pkg="move" type="joy" name="joy"/>
</launch>
